azext_edge/edge/providers/orchestration/resources/connector/opcua/certs.py

```diff
@@ -191,9 +224,12 @@ def client_add(
         application_uri: str,
         public_key_secret_name: Optional[str] = None,
         private_key_secret_name: Optional[str] = None,
+        overwrite_secret: Optional[bool] = False,
```

Comment:
Optional means None is supported, if only true/false you don't need the Optional. If None is supported the default value here can be None.